kipoiseq/dataloaders/sequence.py

```diff
@@ -381,3 +383,52 @@ def get_output_schema(cls):
             output_schema.targets = None
 
         return output_schema
+    
+class AnchoredGTFDl(Dataset):
+    
+    def __init__(self, gtf_file, fasta_file, 
+                 num_upstream, num_downstream,
+                 gtf_filter, anchor_extractor,
+                 transform,
+                 interval_attrs,
+                 use_strand=True):
+        gtf = pr.read_gtf(gtf_file).df
+        gtf = gtf_filter(gtf)  # Optional
```

Comment:
allow gtf_filter to be a string (representing the argument for pd.DataFrame.query) or a function. If None, skip the filtering step.

Reply:
Done